server/audit_math/suite.py

```diff
@@ -133,17 +135,14 @@ def compute_pvalue(
             * np.sum(np.log(self.num_ballots - 2 * Nw + null_margin - np.arange(n_u)))
         )
 
-        upper_n_w_limit = int((self.num_ballots - n_u + null_margin) / 2)
+        upper_n_w_limit = (self.num_ballots - n_u + null_margin) / 2.0
         lower_n_w_limit = np.max([n_w, n_l + null_margin])
 
         # For extremely small or large null_margins, the limits do not
         # make sense with the sample values.
         if upper_n_w_limit < n_w or (upper_n_w_limit - null_margin) < n_l:
             raise ValueError("Null is impossible, given the sample")
 
-        if lower_n_w_limit > upper_n_w_limit:
-            lower_n_w_limit, upper_n_w_limit = upper_n_w_limit, lower_n_w_limit
```

Comment:
how are you sure this is unreachable?

Reply:
Yes. Taken with the statement above it, it is mathematically impossible for one of these ifs to be true when the other isn't. Therefore, if we pass the first if, this one can never happen. I worked out the math, I can take a picture of my notes if you would like!